["Error 3178: subset-05.vdm at 13:14 Right hand of subset is not a set"]